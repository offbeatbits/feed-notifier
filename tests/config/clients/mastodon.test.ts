import dotenv from 'dotenv';
import path from 'path';

describe('Mastodon client config', () => {
  dotenv.config({
    path: path.resolve(process.cwd(), '.env.testing'),
  });

  it('should export Mastodon config', () => {
    expect(require('../../../src/config/clients/mastodon')).toMatchSnapshot(); // eslint-disable-line
  });
});
