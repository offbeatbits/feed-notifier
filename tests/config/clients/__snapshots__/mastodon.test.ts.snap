// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mastodon client config should export Mastodon config 1`] = `
Object {
  "mastodon": Object {
    "accessToken": "test_mastodon_access_token",
    "key": "test_mastodon_client_key",
    "secret": "test_mastodon_client_secret",
  },
}
`;
